scripts/deployment/deployProd.ts

```diff
@@ -34,15 +34,15 @@ const optimismSepoliaChainConfiguration = {
   },
 }
 
-const ecoTestNetChainConfiguration = {
```

Comment:
are the preprod and prod deploys sufficiently different to warrant having a separate script? shouldnt they be basically identical?